channels/pe.m3u

```diff
@@ -51,8 +51,10 @@ https://stmv.panel.mivideo.pro/mileniuntv/mileniuntv/playlist.m3u8
 http://vs8.live.opencaster.com/20100152275/jcpstream/playlist.m3u8
 #EXTINF:-1 tvg-id="OndaDigital.pe" tvg-name="Onda Digital" tvg-country="PE" tvg-language="" tvg-logo="https://i.imgur.com/3R2emx7.jpg" group-title="Music",Onda Digital
 http://184.154.28.210:1935/ondadigital/ondadigital/playlist.m3u8
-#EXTINF:-1 tvg-id="PanamericanaTV.pe" tvg-name="Panamericana TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uQhEDES.png" group-title="",Panamericana TV [Not 24/7]
+#EXTINF:-1 tvg-id="PanamericanaTV.pe" tvg-name="Panamericana TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uQhEDES.png" group-title="",Panamericana TV (HD) [Not 24/7]
 https://cdnhd.iblups.com/hls/ptv2.m3u8
+#EXTINF:-1 tvg-id="PanamericanaTV.pe" tvg-name="Panamericana TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uQhEDES.png" group-title="",Panamericana TV (SD) [Not 24/7]
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/PanamericanaPTV
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/120113367-d5189780-c182-11eb-8676-e9566b17a4c1.png)

Reply:
This issue is the scope of @LaneSh4d0w

Reply:
Ouch. I really need to see why it blocks like that over time.

Reply:
It is now operational again, however, I am not sure whether to tag [Not 24/7].

Reply:
I miiiiight know why it randomly crashes. Someone is literally reloading my server for one SINGLE Dailymotion channel every 3 seconds.